### ISSUE TITLE
No promo code

### ISSUE THREAD

**Issue by GabiTM2** [ChaseSA/Roobet-Player#1]
Hello !
Your script looks awesome ! 
But i'm looking for a promo code but the only one i found doesn't give any points.
Do you have a promo code to provide that gives points to play ?

**Comment by GabiTM2** [comment]
Also, i have this error when i try to launch the program : 
![image](https://user-images.githubusercontent.com/64553368/98802085-0910b300-2413-11eb-83da-91570a92a63f.png)

**Comment by ChaseSA** [comment]
> Also, i have this error when i try to launch the program

Hello there, this error is because you're missing the `config.json` file. This script utilizes the [2captcha](url) API for solving captchas on registration so you need to input your API key. Here is the format
`{ "captcha-key":"put_key_here"}`
You will also need to create a `proxies.txt` file and input your proxies in ip:port format as Roobet has ratelimiting

**Comment by GabiTM2** [comment]
Hi !
I created the proxies.txt and the config.json file but I had not filled in the file the right way.
Also how is it working with promo codes ? Because they are only providing a daily rakeback bonus...
Do you have discord in order to discuss more easily?

Ty for your help! 
Cordially

**Comment by ChaseSA** [comment]
Yes, leave your Discord down below and I can add you tomorrow when I am online

**Comment by GabiTM2** [comment]
Just add Gabi#0003

**Comment by ChaseSA** [comment]
Added you

**Comment by jaugustin12** [comment]
I am interested in this project as well. My discord is jba5337#5296
